packages/atomic/src/components/search/atomic-result-badge/atomic-result-badge.new.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type {Meta, StoryObj as Story} from '@storybook/web-components-vite';
+import {getStorybookHelpers} from '@wc-toolkit/storybook-helpers';
+import {MockSearchApi} from '@/storybook-utils/api/search/mock';
+import {parameters} from '@/storybook-utils/common/common-meta-parameters';
+import {wrapInResultList} from '@/storybook-utils/search/result-list-wrapper';
+import {wrapInResultTemplate} from '@/storybook-utils/search/result-template-wrapper';
+import {wrapInSearchInterface} from '@/storybook-utils/search/search-interface-wrapper';
+
+const searchApiHarness = new MockSearchApi();
+
+searchApiHarness.searchEndpoint.mock((response) => ({
+  ...response,
+  results: response.results.slice(0, 1),
+  totalCount: 1,
+  totalCountFiltered: 1,
+}));
+
+const {events, args, argTypes, template} = getStorybookHelpers(
+  'atomic-result-badge',
+  {excludeCategories: ['methods']}
+);
+
+const {decorator: searchInterfaceDecorator, play} = wrapInSearchInterface({
+  includeCodeRoot: false,
+});
+const {decorator: resultListDecorator} = wrapInResultList('list', false);
+const {decorator: resultTemplateDecorator} = wrapInResultTemplate();
+
+const meta: Meta = {
+  component: 'atomic-result-badge',
+  title: 'Search/Result Badge',
+  id: 'atomic-result-badge',
+  render: (args) => template(args),
+  decorators: [
+    resultTemplateDecorator,
+    resultListDecorator,
+    searchInterfaceDecorator,
+  ],
+  parameters: {
+    ...parameters,
+    msw: {
+      handlers: [...searchApiHarness.handlers],
+    },
+    actions: {
+      handles: events,
+    },
+  },
+  args,
+  argTypes,
+  play,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'Using a field',
+  args: {field: 'title'},
+};
+
+export const WithLabel: Story = {
+  name: 'Using a label',
+  args: {label: 'Trending'},
+};
+
```

Comment:
Should we also have stories showcasing scenarios as in the component's JSDoc?

i.e.,

```
 * * Slotted elements:
 * ```html
 * <atomic-result-badge icon="https://my-website.fake/stopwatch.svg">
 *     Deal ends in <my-dynamic-countdown></my-dynamic-countdown>
 * </atomic-result-badge>
 * ```
 *
 * The contents of a multi-value field can be displayed as in the following example:
 * ```html
 * <atomic-result-badge icon="https://my-website.fake/language.svg">
 *    <atomic-result-multi-value-text field="language"></atomic-result-multi-value-text>
 * </atomic-result-badge> 
```